ios-nocodesign-13-2-dep-9-3.cmake

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2017-2019, Ruslan Baratov
+# All rights reserved.
+
+if(DEFINED POLLY_IOS_NOCODESIGN_13_2_DEP_9_3_CMAKE_)
+  return()
+else()
+  set(POLLY_IOS_NOCODESIGN_13_2_DEP_9_3_CMAKE_ 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_clear_environment_variables.cmake")
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_init.cmake")
+
+set(IOS_SDK_VERSION 13.2)
+set(IOS_DEPLOYMENT_SDK_VERSION 9.3)
+
+set(POLLY_XCODE_COMPILER "clang")
+polly_init(
+    "iOS ${IOS_SDK_VERSION} / Deployment ${IOS_DEPLOYMENT_SDK_VERSION} / arm64 / \
```

Comment:
Not an `arm64` toolchain, it's universal, e.g.:
- https://github.com/ruslo/polly/blob/31dd6a7d4beda083b1912347ad52bf2288fc53c2/ios-nocodesign-11-2.cmake#L17